docs/search.rst

```diff
@@ -0,0 +1,23 @@
+Search
+=============
+
+There are two possible mechanisms supporting search in saleor. PostgreSQL full text search capability can be used or external Elasticsearch service.
+
+PostgreSQL full text search
+---------------------------
+
+PostgreSQL search in enabled by default, see :doc:`postgresql_search` for more information.
+
+
+Elasticsearch
+-------------
+
+You can integrate saleor with Elasticsearch service, see :doc:`elasticsearch` for more.
```

Comment:
The word "Saleor" should be capitalized here (and in few other places below as well).